docs/standard/glossary.md

```diff
@@ -35,6 +35,12 @@ A cross-platform, high-performance, open source implementation of ASP.NET built
 
 See [ASP.NET Core](/aspnet/#pivot=core).
 
+## assembly
+
+A *.dll* file that contains a collection of APIs that can be called by apps or other assemblies.
```

Comment:
I often hear folks referring to the *.dll* files in the bin folder as their project's binaries after they've built a project in VS.